windows/win_file.py

```diff
@@ -56,17 +56,26 @@
 
 EXAMPLES = '''
 # create a file
-- win_file: path=C:\\temp\\foo.conf 
+- win_file:
+    path: C:\temp\foo.conf
 
 # touch a file (creates if not present, updates modification time if present)
-- win_file: path=C:\\temp\\foo.conf state=touch 
+- win_file:
+    path: C:\temp\foo.conf
```

Comment:
Using the Windows modules can you write directly to network shares `//server/foo/bar`, if so is anything needed to escape that correctly?